### PR TITLE
Updated core docs

### PR DESCRIPTION
**Pull request by srihari-humbarwadi-intel**

### CODE REVIEW

File: docs/core/usage.md [hunk 1]
```diff
@@ -5,36 +5,28 @@ The SDK is designed to support a broad spectrum of public models available on th
 
 The subsequent sections provide a detailed overview of the SDK API usage.
 
-### Initializing the Inference Session
-An instance of `InferenceModel` can be created using the `InferenceSession.create_session` method.
+### Loading the Desired Model
```
Comment (ck-intel) [inline docs/core/usage.md:8]: Loading the desired model and Specifying the inference backends can be merged as the underlying API is same, and in loading also a default backend is used.

File: docs/core/usage.md [hunk 2]
```diff
@@ -5,36 +5,28 @@ The SDK is designed to support a broad spectrum of public models available on th
...
-### Loading the Desired Model
-The `InferenceModel.prepare_model` method is used to load the `microsoft/phi-2` model from the Hugging Face hub.
-> Note: This action triggers a download of model files during the first run. Subsequent runs will utilize locally cached files to avoid repeated downloads.
-
-```python
-session.prepare_model(model_name='microsoft/phi-2')
+model = InferenceModel.init(model_name='microsoft/phi-2', inference_backend='pytorch')
 ```
 
 #### Loading Quantized Weights (INT8)
```
Comment (ck-intel) [inline docs/core/usage.md:24]: "Prepare" API description is missing, which can be used to compress OV model to int8.

**Review by ck-intel — CHANGES_REQUESTED** [review]

File: docs/core/usage.md (hunk 1 above)
Comment (srihari-humbarwadi-intel) [reply docs/core/usage.md:8]: its a subsection under `Loading the Desired Model`

**Review by ck-intel — APPROVED** [review]
LGTM

**Pull request merged**